db/migrate/20140712034016_add_timestamps_to_quotes.rb

```diff
@@ -1,7 +1,7 @@
 class AddTimestampsToQuotes < ActiveRecord::Migration
   def change
     change_table :quotes do |t|
-      t.timestamps
+      t.timestamps null: false
```

Comment:
Hey, this is the right idea -- but this migration has already run in production so to make the database accept the new constraint we actually need to create a new migration, otherwise rails will just assume this one has run, and not change the database.